package-glue-job.sh

```diff
@@ -47,45 +47,37 @@ then
     pipenv requirements > requirements.txt
 
     echo "Installing dependencies..."
-    pip install -q -r requirements.txt --no-deps -t .
+    python -m pip install -q -r requirements.txt --no-deps -t .
 else
     echo "No Pipfile found, assuming no external dependencies are being used."
 fi
 
 # Create Wheel Package
 
-echo "* Creting setup.py file"
-cat >setup.py <<EOL
-from setuptools import setup, find_namespace_packages
-import glob, os
-
-# The the current filename without the extension.
-def get_current_py_module():
-    current_file_name = os.path.basename(__file__)
-    return os.path.splitext(current_file_name)[0]
+echo "* Creating Wheel package"
 
+read -r -d '' CMD <<EOF
+from setuptools import setup, find_namespace_packages
+from glob import glob
+from os.path import basename, splitext
 
-# Returns the filename of other .py files in the current directory without the extensions.
+# Returns the filename of all .py files in the current directory without their extensions.
 def find_py_modules():
-    cwd = os.getcwd()
```

Comment:
Should not be needed. This script is only invoked from the folder in which the relevant files are. It works without the `cwd` stuff.